collab-document/src/document.rs

```diff
@@ -41,8 +41,12 @@ impl Document {
       let txn = &collab_guard.transact();
       collab_guard.get_map_with_txn(txn, vec![ROOT])
     };
+
     match is_document_exist {
-      Some(_) => Ok(Document::get_document_with_collab(collab)),
```

Comment:
The name of the 'get_document_with_collab' function is weird. How about 'open_document_with_collab' or simply 'init_with_collab'?